README.md

```diff
@@ -1 +1,25 @@
-# Hello-Microverse-Project
\ No newline at end of file
+
+# Hello Microverse Project
+ In this Project, I performed a Hello Microverse repository, with all the linters check !
+
+# Prerequisites
+    Setup Install Usage Run tests Deployment Authors
+
```

Comment:
- [ ] Please add `Getting Started` section which contains code to run project locally, For ex:-
- `git clone https://github.com/YannickZahinda/Hello-Microverse-Project.git`
- `cd Hello-Microverse-Project`